docs/api.md

```diff
@@ -220,6 +220,9 @@ Indicates that the browser is connected.
     - `password` <[string]>
   - `colorScheme` <"light"|"dark"|"no-preference"> Emulates `'prefers-colors-scheme'` media feature, supported values are `'light'`, `'dark'`, `'no-preference'`. See [page.emulateMedia(options)](#pageemulatemediaoptions) for more details. Defaults to '`light`'.
   - `logger` <[Logger]> Logger sink for Playwright logging.
+  - `trace` <[Object]> Configures tracing for this context.
+    - `path` <[string]> Path to the trace file.
+    - `resourcesPath` <[string]> Path to the shared resources directory. Multiple traces are encouraged to share the resources directory to improve tracing performance.
```

Comment:
- Let's be consistent and only pass paths for artifact collection into `launch`/`launchPersistent`. Call it `tracesPath`.